docs/index.rst

```diff
@@ -24,6 +24,7 @@ Contents
     resources
     modules
     release
+    viper_support
```

Comment:
I don't see a corresponding `docs/viper_support.rst` file.  Did it not get committed.

Reply:
Woops, fixing!

Reply:
@pipermerriam added the file now :) sorrry again!

Reply:
@jacqueswww I still can't see it. :(

Reply:
@veox try now, must've been a force push fail :(